### PR TITLE
Feature/daef 515 instructions for setting up NTP on Daedalus website

### PR DESCRIPTION
**Pull request by tomothespian**
This PR introduces instructions for setting up NTP on Daedalus website.

**TODOs:**
- [x] Temporarily hide "See problem solutions" button as there is no web link ready
- [x] Implement acceptance-test for this new feature
- [x] Add 2 seconds polling for isSystemTimeCorrect Api call
- [x] Integrate Api call and remove hardcoded "isSystemTimeCorrect" value from NetworkStatusStore
- [x] Inject missing Japanese translations:
```
"systemTime.error.overlayButtonLabel": "!!!See problem solutions",
"systemTime.error.overlayText": "!!!ATTENTION: Time of your machine is different from global time. You are 2 hours 12 minutes 54 seconds behind. You need to fix issue, because you are gonna be unable to sync system!",
"systemTime.error.overlayTitle": "!!!Daedalus Sync Error",
```

systemtimeerroroverlay

### CODE REVIEW

**Comment by nikolaglumac** [comment]
UI part looks great @tomothespian!
We need to wait for Api integration in order to merge this PR...

**Comment by nikolaglumac** [comment]
@darko-mijic we need a URL to point the "See problem solutions" button to. Thanks for providing it!

**Comment by nikolaglumac** [comment]
@darko-mijic @DominikGuzei @tomothespian in the scope of this PR the Api call for checking user's local time sync is implemented as a "slow" poller which is triggered on the application start and in 24h cycles after that. I'm afraid this is not enough. There are two different situations where this would break:
1. If during initial application start time check there is time difference detected and user is presented with the NTP notification, even if the user fixes the local time Daedalus will remain blocked with the notification (as it is not aware of this change - will take another 24h or a manual application restart for the change to be noted)
2. If during normal application usage user on purpose changes the local time Daedalus will not be aware of this and that would probably result in many errors.
...both of this cases point to a need of polling local-time-check call more often - probably every few seconds or so.
We should discuss this.

**Comment by nikolaglumac** [comment]
@darko-mijic @DominikGuzei this PR is ready for review ;)

**Comment by DominikGuzei** [comment]
@nikolaglumac what about the merge conflicts?

**Comment by nikolaglumac** [comment]
@DominikGuzei master has etc-client stuff now. We will have to fix the conflicts obviously...

**Comment by tomothespian** [comment]
@DominikGuzei @nikolaglumac latest master merged.

File: app/stores/NetworkStatusStore.js [hunk 1]
```diff
@@ -199,6 +231,19 @@ export default class NetworkStatusStore extends Store {
     }
   };
 
+  _redirectToSyncingWhenLocalTimeDifferent = () => {
+    if (
+      this.localTimeDifference > this.ALLOWED_TIME_DIFFERENCE &&
+      !this.isSynced &&
+      this.stores.app.currentRoute !== ROUTES.PROFILE.LANGUAGE_SELECTION &&
+      this.stores.app.currentRoute !== ROUTES.PROFILE.TERMS_OF_USE &&
+      this.stores.app.currentRoute !== ROUTES.PROFILE.SEND_LOGS
```
Comment (DominikGuzei) [inline app/stores/NetworkStatusStore.js]: I think we should move these "if we are not on one of the setup pages" checks into a dedicated, reusable function. We have that already in a few places as far as i remember

File: app/api/ada/index.js [hunk 2]
```diff
@@ -675,6 +678,18 @@ export default class AdaApi {
       throw new GenericApiError();
     }
   }
+
+  async getLocalTimeDifference(): Promise<GetLocalTimeDifferenceResponse> {
+    Logger.debug('AdaApi::getWallets called');
```
Comment (DominikGuzei) [inline app/api/ada/index.js]: Copy & paste sin! 😆

File: app/api/ada/index.js [hunk 3]
```diff
@@ -675,6 +678,18 @@ export default class AdaApi {
...
+    try {
+      const response: AdaLocalTimeDifference = await getAdaLocalTimeDifference({ ca });
+      Logger.debug('AdaApi::getLocalTimeDifference success: ' + stringifyData(response));
+      return response;
+    } catch (error) {
+      Logger.error('AdaApi::getWallets error: ' + stringifyError(error));
```
Comment (DominikGuzei) [inline app/api/ada/index.js]: Here too 😉

File: app/api/ada/mocks/patchAdaApi.js [hunk 4]
```diff
@@ -36,4 +38,13 @@ export default (api: AdaApi) => {
     }
     return { amount: new BigNumber(1000) };
   };
+
+  api.getLocalTimeDifference = async () => {
+    const response = Promise.resolve(LOCAL_TIME_DIFFERENCE);
+    return response;
```
Comment (DominikGuzei) [inline app/api/ada/mocks/patchAdaApi.js]: This function can be shortened to a one-liner

File: app/components/loading/SystemTimeErrorOverlay.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        <SvgInline svg={attentionIcon} className={styles.icon} />
+
+        <h1>{intl.formatMessage(messages.overlayTitle)}</h1>
+
+        <p>{intl.formatMessage(messages.overlayText, { behindTime })}</p>
+
+        {/* <Button
+          label={intl.formatMessage(messages.buttonLabel)}
+          skin={<SimpleButtonSkin />}
+        /> */}
```
Comment (DominikGuzei) [inline app/components/loading/SystemTimeErrorOverlay.js]: Why is this button commented out?

File: app/api/ada/index.js (hunk 2 above)
Comment (tomothespian) [reply app/api/ada/index.js]: aaah, my bad hehe 😆

File: app/components/loading/SystemTimeErrorOverlay.js (hunk 5 above)
Comment (nikolaglumac) [reply app/components/loading/SystemTimeErrorOverlay.js]: This was requested by @darko-mijic...

**Pull request merged**